client/assets/stylesheets/_calypso-color-scheme-jetpack-cloud.scss

```diff
@@ -1,5 +1,5 @@
 // Jetpack Cloud color scheme for Calypso
-.color-scheme.is-group-jetpack-cloud {
+.is-group-jetpack-cloud {
```

Comment:
Can't just use root here instead?

Reply:
I wouldn't do that. The [default color scheme already uses a `:root` pseudo-element selector](https://github.com/Automattic/wp-calypso/blob/a8271b2e532f634da94f751cf9adc15e491328c4/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss#L1) which brings a potential specificity issue here.

Actually, after a deeper investigation, the current approach may result in a specificity issue, too. [The pseudo-element selector has the same specificity as the class selector](https://specifishity.com/). In other color schemes there are always two class selectors used so that the specific color scheme always "wins", e.g.:

```css
.color-scheme.is-contrast
```

However, in our approach there's just one class selector. It means that the order in which CSS files are loaded determines which color scheme is used.

I'm going to increase the specificity here.

Reply:
Addressed in 7c4d54f.